src/plugins/safeSnap/utils/umaModule.ts

```diff
@@ -108,7 +110,9 @@ export const getModuleDetailsUma = async (
         toUtf8Bytes(proposalHash.replace('0x', ''))
       ]
     );
-    timestamp = await moduleContract.proposalHashes(proposalHash);
+    currentProposalHashTimestamp = await moduleContract.proposalHashes(
```

Comment:
Is `currentProposalHashTimestamp` used anywhere or can it be removed? It looks like `proposalHashTimestamp` is used below to retrieve active proposals.

Reply:
Good catch. That was part of an earlier iteration that tried to identify the current proposal hash timestamp, if any, but it was less efficient.